test/conformance/source/environment.cpp

```diff
@@ -62,6 +97,10 @@ std::ostream &operator<<(std::ostream &out,
 uur::PlatformEnvironment::PlatformEnvironment(int argc, char **argv)
```

Comment:
This is a pretty meaty function doing a lot, maybe it could be split up to simplify?